exercise9.rb

```diff
@@ -0,0 +1,33 @@
+require 'selenium-webdriver'
+
+class Exercise
+
+    def setup
+        Selenium::WebDriver::Firefox::Service.driver_path = "/home/vibhav/Desktop/INFUSE/driver/geckodriver"
+        @driver = Selenium::WebDriver.for :ff
+        @url = "https://cosmocode.io/automation-practice-webtable"
+    end
+
+    def action
+        begin
+            @driver.get(@url)
+           
+            select = @driver.find_element(:tag_name, "tr")
+            #puts select
+            all_headers = select.find_elements(:tag_name, "strong")
```

Comment:
Instead of doing this, get the first row of the table, thats your header

Reply:
Updated code as per the comment